ethereum/truffle-config.js

```diff
@@ -311,6 +311,16 @@ module.exports = {
       },
       network_id: 421613,
     },
+    arbitrum_sepolia_testnet: {
+      provider: () => {
+        return new HDWalletProvider(
+          process.env.MNEMONIC,
+          "https://arbitrum-sepolia.publicnode.com"
+          //"https://sepolia-rollup.arbitrum.io/rpc" // This didn't work.
```

Comment:
```suggestion
```

Reply:
I would like to leave the two that say "This didn't work" because they are the ones listed in the official docs. I think someone might wonder why I'm using something other than the documented ones.